lib/routes/home/widgets/bottom_actions_bar/bottom_actions_bar.dart

```diff
@@ -26,7 +26,7 @@ class BottomActionsBar extends StatelessWidget {
     return BlocBuilder<AccountBloc, AccountState>(
       builder: (context, account) {
         return BlocBuilder<LSPBloc, LspState?>(builder: (context, lspState) {
-          final connected = lspState?.lspInfo != null;
+          final selectedLsp = (lspState?.selectedLspId != null);
```

Comment:
I think `isLspSelected` is a better suited name for this variable, in all instances it's used.